src/react/Virtualizer.tsx

```diff
@@ -283,7 +283,7 @@ export const Virtualizer = forwardRef<VirtualizerHandle, VirtualizerProps>(
         resizer.$observeRoot(e);
         scroller.$observe(e);
       };
-      if (scrollRef) {
+      if (scrollRef && scrollRef[refKey]) {
         // parent's ref doesn't exist when useLayoutEffect is called
         microtask(() => assignScrollableElement(scrollRef[refKey]!));
```

Comment:
@m-a-r-c-e-l-i-n-o 
Hi, thank you for the PR!

I couldn't reproduce the issue, but anyway this fix is problematic. The microtask is necessary to assign parent element to virtualizer, so it may solve the issue but also break virtualization.

Could you try the following fix instead?
```suggestion
      if (scrollRef) {
// parent's ref doesn't exist when useLayoutEffect is called
microtask(() => {
  if (scrollRef[refKey]) {
    assignScrollableElement(scrollRef[refKey]!)
  }
});
```

Reply:
>  It prevents the error from coming up but still freezes/crashes the browser.

Could you explain it in detail? If we do early-return inside of microtask, the microtask does nothing. So freezing will not happen in theory.